test/e2e/framework/util.go

```diff
@@ -505,6 +505,7 @@ func LoadClientset() (*clientset.Clientset, error) {
 
 // RandomSuffix provides a random sequence to append to pods,services,rcs.
 func RandomSuffix() string {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
this seems like a very unusual approach to seeding.
normally a seed is supplied once and used for the duration, this allows overriding it to be reproducible.